### ISSUE TITLE
Recommend native overlayfs instead of fuse-overlayfs?

### ISSUE THREAD

**Issue by StarpTech** [containers/podman#15461]
Hi, according to https://www.redhat.com/sysadmin/podman-rootless-overlay is it still recommended to use `mount_program = "/usr/bin/fuse-overlayfs"` when Kernel 5.13 has shipped native support?

https://github.com/containers/podman/blob/main/contrib/podmanimage/stable/Containerfile#L38-L42

**Comment by mheon** [comment]
I don't think that blog says we still recommend `fuse-overlayfs` when native Overlay is available; unless you have a good reason to continue to use fuse-overlay (it does some things we don't have kernel support for, at least last I checked; UID mapping would be the most notable).

Commenting the `mount_program` line will indeed be sufficient to swap the driver.
